modules/apps/29-fee/types/msgs.go

```diff
@@ -82,12 +84,12 @@ func (msg MsgPayPacketFee) ValidateBasic() error {
 
 	// if any of the fee's are invalid return an error
 	if !msg.Fee.AckFee.IsValid() || !msg.Fee.ReceiveFee.IsValid() || !msg.Fee.TimeoutFee.IsValid() {
-		return sdkerrors.ErrInvalidCoins
+		return ErrInvalidFee
```

Comment:
Is there a need to add a new error type? What about wrapping the `ErrInvalidCoins` with additional info? 

eg. `sdkerrors.Wrap(sdkerrors.ErrInvalidCoins, "incentivized packet contains one or more invalid fees")`

Reply:
I like @damiannolan suggestion